includes/discovery/sensors/count/cisco.inc.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/*
+ * LibreNMS - Cisco count sensors
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ *
+ * @package    LibreNMS
+ * @subpackage webui
+ * @link       http://librenms.org
+ * @copyright  2021 LibreNMS
+ * @author     LibreNMS Contributors
+*/
+
+$tables = [
+    ['num_oid' => '.1.3.6.1.4.1.9.9.661.1.3.1.1.1.',    'oid' => 'c3gImsi',                   'state_name' => 'c3gImsi',                    'mib' => 'CISCO-WAN-3G-MIB', 'descr' => 'IMSI'],
```

Comment:
Hi
IMSI and IMEI are not something you want to store in an RRD. You should use it in the description of other sensors, or as a group name, in order to see it without wasting ressources to store it with historical data every 5 minutes.
Nearby cells are completely OK, they should remain in Count sensor.

Reply:
@PipoCanaja thanks for the feedback. IMEI is indeed mostly static, IMSI graphing has in our environment great use as they can change more often. Nevertheless I understand your feeling about these sensors. The thing is that I do think the presentation now is very clean, and your suggestions to safe some cycles is to do the presentation in a dirty way and put data on places which are not made for it. I sincerely do not think that would be the right way to go. In general I think what we are mostly missing is a sensor of type string that will not be graphed. Would be a really nice addition and would also be a clean way of presenting other data than numerical values. As this would be a core enhancement, i don't think I should work in that direction. I will think a bit more about an alternatives for IMEI and IMSI.

Reply:
@Schouwenburg I understand your need, but RRDs are definitely not the way to go, as you know. We don't currently have an ideal solution to store historical data for a sensor outside an RRD. Eventlog is the only existing structure that would allow you to keep history, and the data itself could be stored in components. They are completely flexible.
Only current IMSI/IMEI would be in the SQL db, and the historical data would be in the eventlog (at every poll, if new value is different from current one, you generate an eventlog in order to keep track).

Reply:
@PipoCanaja Ok, as per your request IMSI and IMEI are now removed as sensors and added to entity-physical.